hetmatpy/pipeline.py

```diff
@@ -12,6 +12,14 @@
 FLOAT_ERROR_TOLERANCE = 1e-5
 
 
+def sd_is_positive(sd):
+    """
+    Tests whether the standard deviation is greater than zero or if it is
+    zero/NaN/None
+    """
+    return sd > 0 and sd is not None and sd != numpy.nan
```

Comment:
use `numpy.isnan` or `pandas.isna`. The nice thing about `pandas.isna` is that it returns True for None or numpy.nan.